packages/react/src/field/FieldMessage.tsx

```diff
@@ -1,14 +1,14 @@
+import { PropsWithChildren } from 'react';
 import { Box, Flex } from '../box';
 import { AlertFilledIcon } from '../icon';
 import { Text } from '../text';
 
-export const FieldMessage = ({
-	children,
-	id,
-}: {
-	children: string;
+export type FieldMessageProps = PropsWithChildren<{
```

Comment:
For consistency, I've added prop comments here and exported the type while I was touching these files.